src/client/app/offers/offers.html

```diff
@@ -43,15 +43,27 @@
                 <div class="col-xs-9 col-sm-9 col-md-9 col-lg-9">
                     <div class="date">
                         <span>10/07/2015</span><br>
-                        <span class="activeNeed">Катя</span>
-                        <span class="Need">Єдинорог</span>
+                        </div class="needInfo">
+                        <div>
+                        <div class="activeNeed">Катя</div>
+                         <div class ="offerInfo" ng-hide="editorEnabled">
```

Comment:
unneeded space, active need should also use `{{ }}` syntax instead of hardcoding the value